app/api/admin_statistics_api/users.py

```diff
@@ -53,17 +53,28 @@ def get_all_user_roles(self, role_name):
             Event.deleted_at.is_(None), UsersEventsRoles.role == role)
         return uers
 
+    def get_distinct_roles_count(self, query_result):
+        """
+        Returns the length of set of users for the given query
+        """
+        distinct_entry = set([i.user for i in query_result])
```

Comment:
what is i?

Reply:
@iamareebjamal I will improvise on the variable choice in the next commit